reports/manuscript.tex

```diff
@@ -0,0 +1,19 @@
+\documentclass[11pt]{article} 
```

Comment:
I would say that reports are not needed to be merged in the codebase

Reply:
Yes. I will remove it.